lib/commands/dialog/index.js

```diff
@@ -96,7 +97,12 @@ class DialogCommand extends AbstractCommand {
                     throw `Failed to obtain skill-id from project resource file ${CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG}`;
                 }
             }
-            locale = locale || process.env.ASK_DEFAULT_DEVICE_LOCALE;
+            const skillPackageSrc = ResourcesConfig.getInstance().getSkillMetaSrc(profile);
```

Comment:
If skillId is from the user input, then ask-resources.json might not exist... That's for the use case when users don't have a project locally but doesn't bother them talking to the skill.

This means if skillId is provided by users, then locale value must be provided.